lib/bolt_server/transport_app.rb

```diff
@@ -291,7 +300,16 @@ def make_winrm_target(target_hash)
       }
 
       opts = target_hash.clone.merge(overrides)
-      Bolt::Target.new(target_hash['hostname'], opts)
+
+      data = {
+        'uri' => target_hash['hostname'],
+        'config' => {
+          'winrm' => opts
```

Comment:
Should we have `'transport' => 'winrm'` here like we have for ssh above?

Reply:
For some reason it breaks the tests. Not really sure why...

Reply:
Fixed by setting defaults to `ssl` and `ssl-verify` to `false`.